aws_jupyter_proxy/awsconfig.py

```diff
@@ -0,0 +1,19 @@
+from notebook.base.handlers import IPythonHandler, utcnow
+import json
+from botocore.session import get_session
+
+class AwsConfigHandler(IPythonHandler):
+    def get(self):
+        response = {}
+        self.set_header("Content-Type", "application/json")
+
+        if self.request.uri == "/awsproxy/awsconfig":
+            response = {
+                "region" : self._get_aws_region()
+            }
+
+        self.write(json.dumps(response))
+
+    def _get_aws_region(self):
+        session = get_session()
+        return session.get_config_variable("region") or "us-east-1"
```

Comment:
What is the motivation of keeping `us-east-1` as the default? I'm not sure if that is what every client of this endpoint would what.

Reply:
As discussed, default region will be changed to `None`